app/helpers/shops_helper.rb

```diff
@@ -0,0 +1,2 @@
+module ShopsHelper
```

Comment:
remove this file.
And don't create bu scaffold @AndriiBuhalo !!!

Reply:
okay, done